### DISCUSSION TITLE
Can't Activate Office 365 using Windows 11 in UTM on MacAir M1 2020

### DISCUSSION THREAD

**Discussion by routineCode** [utmapp/UTM#5083] [UTM Help (macOS)]
**Describe the issue**  
I installed the latest version of UTM on a MacAir M1 2020 running Mac OS 13.2.1 and ran Windows 11 without problems. I was also able to install and run Office 365 applications (Word, Excel, pp) without problems. However, the notice said they would only work a few days and then must be activated.

I tried many things to activate Office 365 but nothing worked -- always got a message saying Office could not activate at this time, please try again later. So I contacted Microsoft. They guided me through uninstalling and reinstalling everything, checked network issues in multiple ways using multiple devices and 2 different internet service providers. This same device had no trouble activating and using Office 365 using Parallels instead of UTM (same Windows 11 version). Eventually I gave up, concluding UTM was the problem.

**Configuration**  
all answered above

**Comment by foerdi1** [comment]
Have you solved the problem? I have the same issue :(

**Reply by routineCode** [reply]
No, I have not solved the problem. After many sessions with Microsoft support, we had to give up. Activation of office 365 on Windows under UTM is not possible on my M1 MacAir

**Reply by Satanmat** [reply]
I am having this issue as well.

M2 MBP UTM MacOS 13.3. Win11 Arm can login to 365, on Edge.  but no app Word, or Outlook will sign in. I'm loving the error messages too...
<img width="354" alt="Screenshot 2023-04-06 at 3 47 27 PM" src="https://user-images.githubusercontent.com/65261150/230507565-e8cd8799-92b9-4ed0-aed0-60c36724b100.png">

same server error in Word for two weeks now...

can't even get outlook to connect , firewall on or off...

**Comment by trey-a-12** [comment]
See if you can run and activate Office 365 in the native macOS. I was able to do so in mine via [microsoft.com](url), then install the Office 365 suite again, running it without issue.

**Reply by routineCode** [reply]
Activating office 365 on the Mac OS has never been a problem. But I want to also use Office 365 underWindows since Office 365 on the Mac lacks many features compared to office 365 under Windows 

The problem seems to be specific to UTM since activation has never been a problem with other software, such as Parallels, VMware

**Reply by trey-a-12** [reply]
I certainly agree, as someone who has Office 365 in macOS and Windows, often using the Windows version instead. You say you can't activate it in UTM Windows though? Interesting...
As long as you've installed Windows Guest Tools so you can use internet and all, I'd think you're able to run Office 365 in UTM, and you said you could. Have you logged out and back in to Office 365, after making sure you downloaed the full package from Microsoft's website? I had to download Office 365 Setup via their website to get everything working in mine.
[https://account.microsoft.com/services/microsoft365/details?OCID=PROD_Office_CONS_MyAccount_Migration&openInstall=true&refd=account.microsoft.com](url)
![image](https://user-images.githubusercontent.com/63821515/227580735-8c04536f-8d3c-41ee-9ecd-51036eb7d24c.png)
![Office 365 in UTM](https://user-images.githubusercontent.com/63821515/227581882-50000566-0a76-4296-9e4a-42ebbe7f875c.png)

**Reply by routineCode** [reply]
I have done everything, you stated, and much, much more under the guidance of at least three different Microsoft representatives (the last of whom was obviously very competent). As I said in my original post, there is no problem with the office 365 programs running under windows in UTM -- the problem is that they will not continue to run after a week unless the product is properly activated -- it is only the activation process that is not working. Maybe it's working for others with the same set up and that is part of the reason that I posted here. However, one person (foerdi1) has responded to this thread, indicating that it doesn't work for them either.

**Comment by trey-a-12** [comment]
Sorry to hear that. If nothing else, I’d try installing Windows in another
UTM VM through another ISO, going through setup, signing into Microsoft in
the VM one more time, and trying one last time if you still needed it. If
that still doesn’t work for you, I apologize for wasting your time, and
hope this does get figured out sometime later.

On Fri, Mar 24, 2023 at 11:21 AM Tony ***@***.***> wrote:

> I have done everything, you stated, and much, much more under the guidance
> of at least three different Microsoft representatives (the last of whom was
> obviously very competent). As I said in my original post, there is no
> problem with the office 365 programs running under windows in UTM -- the
> problem is that they will not continue to run after a week unless the
> product is properly activated -- it is only the activation process that is
> not working. Maybe it's working for others with the same set up and that is
> part of the reason that I posted here. However, one person (foerdi1) has
> responded to this thread, indicating that it doesn't work for them either.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/utmapp/UTM/discussions/5083#discussioncomment-5420497>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APG5NS5GEWF7BUDVXR5GZMLW5XCXZANCNFSM6AAAAAAWGXHUPM>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
~ Thomas Aldridge II

**Comment by trey-a-12** [comment]
Have you installed/can you install Office apps through the MS Store? If you
don’t have the MS Store, there’s a command you can run in command prompt,
something like “~i wsreset” or something like that, can’t remember.

On Thu, Apr 6, 2023 at 5:50 PM Satanmat ***@***.***> wrote:

> I am having this issue as well.
>
> M2 MBP UTM MacOS 13.3. Win11 Arm can login to 365, on Edge. but no app
> Word, or Outlook will sign in. I'm loving the error messages too...
> [image: Screenshot 2023-04-06 at 3 47 27 PM]
> <https://user-images.githubusercontent.com/65261150/230507565-e8cd8799-92b9-4ed0-aed0-60c36724b100.png>
>
> same server error in Word for two weeks now...
>
> can't even get outlook to connect , firewall on or off...
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/utmapp/UTM/discussions/5083#discussioncomment-5548587>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APG5NS5CE7Z2PJKMAZNOTX3W75CEHANCNFSM6AAAAAAWGXHUPM>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
~ Thomas Aldridge II

**Reply by Satanmat** [reply]
oh yea... I installed from 365. wiped and reinstalled 2021 LTSC ... anyway... 

found this
https://answers.microsoft.com/en-us/msoffice/forum/all/i-am-unable-to-activate-office-365-arm-on-windows/e152cd3f-48ac-45b7-99f0-f4bb08f25dff?page=1

so It is looking like there is an ARM issue with 365 auth.. everything else seems to work otherwise..

**Reply by routineCode** [reply]
I do not believe it is an ARM issue. Please see my original post where I pointed out that I had no trouble activating office 365 using Parallels and Windows 11 ARM. I did this a year ago and used it for 10 months that way.
